rasa/core/training/dsl.py

```diff
@@ -255,7 +255,9 @@ async def read_from_file(
         """Given a md file reads the contained stories."""
 
         try:
-            with open(filename, "r", encoding=io_utils.DEFAULT_ENCODING) as f:
+            with open(
+                filename, "r", encoding=io_utils.DEFAULT_ENCODING, errors="ignore"
```

Comment:
@evgeniiaraz Why is the `ignore` needed?

Reply:
this is to do with windows and it not always handling all the symbols in utf-8 properly, but we could remove it.